composer.json

```diff
@@ -48,7 +48,7 @@
         "doctrine/annotations": "^1 || ^2",
         "doctrine/coding-standard": "^12",
         "doctrine/deprecations": "^1.0",
-        "doctrine/orm": "^2.14 || ^3.0",
+        "doctrine/orm": "^2.17 || ^3.0",
```

Comment:
Bumping the dev requirement will only affect our CI. You also need to update the conflict rule below.

Reply:
My bad, updated that now